src/org/usfirst/frc/team2175/subsystem/ShooterSubsystem.java

```diff
@@ -71,7 +68,8 @@ public ShooterSubsystem() {
     }
 
     public void setShooterDefaultSpeed() {
-        shooterMotor.set(shooterSpeed);
+        shooterMotor.set(-shooterSpeed);
+        shooterMotorTwo.set(shooterSpeed);
```

Comment:
Motors should have their directions negated at the subsystem level. There should be an easy way to do this - `shooterMotor.reverseOutput(true)` or something.